Ch09/Ch09.md

```diff
@@ -544,7 +544,7 @@ select {
 ```
 
 Because JSON doesn't have a way to express dates natively, we can use the `new Date().getFullYear()` to handle date parsing and extracting of the year portion
-of the date. You can see the projection of the document identifier as well. In addition to the usual JavaScript methods, you also have access to functions defined by RavenDB itself, such as `id` and `year` (instead of the more long-winded JavaScript version). A full list of the functions available for your use can be found in RavenDB's online documentation.
+of the date. You can see the projection of the document identifier as well. In addition to the usual JavaScript methods, you also have access to functions defined by RavenDB itself, such as `id` (for the full list, see the online docs). A full list of the functions available for your use can be found in RavenDB's online documentation.
```

Comment:
The way this is written now, you mention that the full list can be found in the online documentation twice. Maybe cut one of those mentions?